activities_sticky_queues/README.md

```diff
@@ -0,0 +1,48 @@
+# Sticky Activity Queues
+
+This sample is a Python implementation of the [Node "Sticky Workers" example](https://github.com/temporalio/samples-typescript/tree/main/activities-sticky-queues), full credit for the design to the authors of that sample. A [sticky execution](https://docs.temporal.io/tasks#sticky-execution) is a job distribution design pattern where all workflow computational tasks are executed on a single worker. In the Golang Temporal SDK this is explicitly supported via the Session option, but in other SDKs a different approach is required. 
+
+Typical use cases for sticky executions include tasks where interaction with a filesystem is required, such as data processing or interacting with legacy access structures. This example will write text files to folders corresponding to each worker, located in 
+
+This implementation will follow the Node example:
```

Comment:
```suggestion
The strategy is:
```

Concerned that the TypeScript example may change out from under us in the future